pages/projects.tsx

```diff
@@ -66,7 +66,7 @@ const projects = [
       description: "A website for registration and management of lockers, designed with mobile-first functionality. Now maintained by the UVic ECSS.",
       status: "Maintained",
       tech: ["Svelte", "TypeScript", "PostgreSQL", "Docker"],
-      meetingTime: "Undecided",
+      meetingTime: "No Meetings Planned",
```

Comment:
Could "No Meetings Planned" be blue? Just to make it obvious on first glance which meetings are scheduled without reading every project.